deployment/examples/kustomization_storage_account.yaml

```diff
@@ -1,14 +1,19 @@
+# This is an example file to use an Azure Storage Account to export data
```

Comment:
👍 Thank you for this, to have a sample file for consuming tools as example or sample set - I fee like this could directly be used as well. (will check it out)